lib/recovery/recover_lost_jobs.rb

```diff
@@ -0,0 +1,25 @@
+module Recovery
+  class RecoverLostJobs
+    STRATEGIES = [
```

Comment:
Minor: since we only have a pattern of one, could we just repeat the code for the time being? I'm not sure whether future recovery will fit this pattern.